include/ProjectVersion.h

```diff
@@ -54,11 +57,10 @@ class ProjectVersion
 
 private:
 	QString m_version;
-	int m_major;
-	int m_minor;
-	int m_release;
-	QString m_stage;
-	int m_build;
+	int m_major = 0;
+	int m_minor = 0;
+	int m_patch = 0;
+	QStringList m_labels = QStringList();
```

Comment:
You don't need ` = QStringList()` because it will be default-initialized if you do nothing.

Reply:
Wait, C++ handles uninitialized objects nicely but not primitives? Completely opposite to java, weird.